### PR TITLE
[Feature] #93 prepare DictionaryEquipmenMainView Data Binding

### LINKED ISSUE #93: [Feature] DictionaryEquipmenMainView Data Binding
# Please write a story
- 유저는 장비군을 확인할 수 있다
   - 장비군의 Entity data를 불러와서 바인딩 한다

## 🤔 Completion conditions

- [ ] make ViewModel
- [ ] make Repository
- [ ] make UseCase
- [ ]  can fetchImage

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number
🔒 Close #93

## Changes

- Changed files
campair/campair.xcodeproj/project.pbxproj
"campair/campair/Application/Images/EquipmentGroup/\354\235\221\352\270\211_\354\225\210\354\240\204.png"
"campair/campair/Application/Images/EquipmentGroup/\355\205\220\355\212\270.png"
campair/campair/Entity/Dictionary/Content/DictionaryEquipmentContent.swift
campair/campair/Entity/Dictionary/Detail/DictionaryDetailCategory.swift
campair/campair/Entity/Dictionary/Detail/EquipmentList.swift
campair/campair/Entity/Dictionary/Main/DictionaryMainCategory.swift
campair/campair/Entity/Dictionary/Main/DictionaryMainCollection.swift
campair/campair/Json/Editor/Detail/DictionaryMainCollectionData.json
campair/campair/Model/DictionaryRepository.swift
campair/campair/UseCase/DictionaryUseCase.swift
campair/campair/View/ContentView.swift
campair/campair/ViewModel/Dictionary/DictionaryMainViewModel.swift

## New features

- make ViewModel
- make Repository
- make UseCase
- can fetchImage

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

File: campair/campair/Model/DictionaryRepository.swift [hunk 1]
```diff
@@ -6,3 +6,68 @@
...
+    case noData
+}
+
+protocol DictionaryEquipmentsFetchable {
+    func fetchEditorCollection(completion: @escaping (Result<DictionaryMainCollection, Error>) -> Void )
+    func fetchImage(urlString: String, completion: @escaping (Result<Data, Error>) -> Void)
+}
+
+struct DictionaryRepository: DictionaryEquipmentsFetchable {
+    func fetchEditorCollection(completion: @escaping (Result<DictionaryMainCollection, Error>) -> Void ) {
```
Comment (Gobans) [inline campair/campair/Model/DictionaryRepository.swift:30]: Should change name

File: campair/campair/View/ContentView.swift [hunk 2]
```diff
@@ -8,16 +8,39 @@
 import SwiftUI
 
 struct ContentView: View {
```
Comment (Gobans) [inline campair/campair/View/ContentView.swift:10]: this file should not included

**Review by Gobans — APPROVED** [review]
Good! After fix merge it

File: campair/campair/Model/DictionaryRepository.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/Model/DictionaryRepository.swift:30]: thank you I changed it

File: campair/campair/View/ContentView.swift (hunk 2 above)
Comment (rriver2) [reply campair/campair/View/ContentView.swift:10]: I deleted it sorry I'll check two time next PR! thank you goban 🌷

File: campair/campair/View/ContentView.swift [hunk 3]
```diff
@@ -8,16 +8,39 @@
...
             }
         }
+        .onAppear {
+            // testcode
+            viewModel.viewAppeared()
+        }
+    }
+}
+
+struct LxtractedView: View {
```
Comment (sustainable-git) [inline campair/campair/View/ContentView.swift]: In swift, we don't usually abbreviate

**Review by sustainable-git — COMMENTED** [review]
GAZAMI

File: campair/campair/View/ContentView.swift (hunk 3 above)
Comment (rriver2) [reply campair/campair/View/ContentView.swift]: I changed this too ! thank you

**Pull request merged**